gensec/customPrecon.py

```diff
@@ -48,8 +49,8 @@ def Kabsh_rmsd(atoms, initial, molindixes, removeHs=False):
     coords1 = np.dot(coords1, K)
     rmsd_kabsh = 0.0
     for v, w in zip(coords1, coords2):
-        rmsd_kabsh += sum([(v[i] - w[i])**2.0 for i in range(len(coords1[0]))])
-    return np.sqrt(rmsd_kabsh/len(coords1))
+        rmsd_kabsh += sum([(v[i] - w[i]) ** 2.0 for i in range(len(coords1[0]))])
+    return np.sqrt(rmsd_kabsh / len(coords1))
 
 
 #    VALUES for ALPHA-vdW and C6_vdW:
```

Comment:
This comment should not be here. Any actual relevant information that is included in this comment should be summarized in the docstring.